site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -127,29 +126,25 @@ Recently released in the Early Access build, and coming soon to Mainline, is sup
-being far slower for emulators that have to constantly intercept, decode and recompile shaders on the fly. 
+A couple decades ago, there was no common language for the newly added programable shading units in GPUs, so the `OpenGL Architecture Review Board` decided to create a proper standardised shading language they called `GLASM`. In broad terms, this is an assembly language used to communicate with the GPU. This makes it very difficult to work with, and the difficulty is only exacerbated by the limited set of debugging tools available. Furthermore, the language was developed with the hardware limitations of the time in mind.
+In the present, `GLASM` has been mostly deprecated in favour of easier-to-work-with, high level shader representations like GLSL or SPIR-V.
+While this means faster results for game developers due to less time spent looking at the code, it also has the disadvantage of being far slower for emulators that have to constantly intercept, decode and recompile shaders on the fly. 
 &nbsp;
 
-In the beggining, support for GLASM started as just an experiment. Armed with [apitrace](https://apitrace.github.io/) as his only debug tool, [Rodrigo](https://github.com/ReinUsesLisp) set to his task. 
-Luckily, and for no apparent sane reason, since Nvidia decided to maintain support for such an old feature even on the latest OpenGL versions, this feature soon became a reality and with this initial [assembly shading](https://github.com/yuzu-emu/yuzu/pull/3964) support in place, Nvidia OpenGL users can enjoy extremely fast shader compilation times. 
+In the beggining, support for `GLASM` started as just an experiment. Armed with [apitrace](https://apitrace.github.io/) as his only debug tool, [Rodrigo](https://github.com/ReinUsesLisp) set to his task. 
+Luckily, and for no apparent sane reason, since Nvidia still maintains support for such an old feature, even on the latest OpenGL versions. As such, support for `GLASM` soon became a reality and with this initial [assembly shading](https://github.com/yuzu-emu/yuzu/pull/3964) support in place, Nvidia OpenGL users can enjoy extremely fast shader compilation times.
 &nbsp;
 
 Due to being closer to the native hardware of the Nintendo Switch, we can also expect some precision fixes, with more coming in the future.
 &nbsp;
 
-Unfortunately, GLASM has some limitations. To list some of them:
+Unfortunately, `GLASM` has some limitations. To list some of them:
 
-- This is an Nvidia only and OpenGL only feature, other vendors (AMD and Intel) offer support for assembly shaders only for the feature sets of the old games that used to require it. This is very unlikely to change in the future.
+- This is an Nvidia only and OpenGL only feature — other vendors (AMD and Intel) only offer support for the specific assembly shaders that old games require and this is highly unlikely to change in the future.
```

Comment:
Forgot to change `Nvidia only and OpenGL only feature` to `Nvidia and OpenGL only feature`